specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +555,183 @@
           }
         }
       }
+    },
+    "/weather/historical/actuals/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Historical Actuals**\n  \n  \n**Applies to**: G1 and G2 pricing tiers.\n\n\nThe service returns climatology data such as past daily actuals for a given coordinate location.",
```

Comment:
To be consistent with how all the Azure Maps Docs pages are going to be updated soon, please change:

G1 and G2 pricing tiers.

To:

Gen 1 (S0 and S1) and Gen 2 pricing tiers